bin/generate-gutenboarding-design-thumbnails.js

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env node
+/**
+ * Generate static design screenshot thumbnails for Gutenboarding
+ * Until we can generate tall screenshots on the fly with mshots:
+ * https://github.com/Automattic/mShots/issues/16
+ * https://github.com/Automattic/wp-calypso/issues/40564
+ *
+ * You'll need capture-website and sharp:
+ *
+ * npm i capture-website sharp
+ *
+ * (I didn't want to add these to dependencies just for this script since these are kinda heavy ones)
+ * https://www.npmjs.com/package/capture-website
+ * https://www.npmjs.com/package/sharp
+ */
+
+const captureWebsite = require( 'capture-website' );
+const designs = require( '../client/landing/gutenboarding/available-designs.json' );
```

Comment:
Will need a bit of work once https://github.com/Automattic/wp-calypso/pull/40587 merges but not a biggie.

Reply:
Oh… that's going to conflict with some changes in https://github.com/Automattic/wp-calypso/pull/40680 (json -> ts)

Reply:
This script is so throwaway material that it doesn't matter. We might need to run this just once or twice more (hope not at all) so even just listing designs here manually would be fine.

(designs = URLs like `https://public-api.wordpress.com/rest/v1/template/demo/mayland/mayland2/?font_headings=Cabin&font_base=Raleway`)

Reply:
We concluded via another channel that this can be addressed when the need arises.